### ISSUE TITLE
Need a Loading (busy indicator) feature in IgrFinancialChart for reactJS

### ISSUE THREAD

**Issue by sagarnanera** [IgniteUI/igniteui-react#14]
Hello There...

I am working on a project on crypto currencies paper trading and I am using MERN stack. In which, I am using IgniteUI library for IgrFinancialChart. The problem is when the data is being fetched from the API (Binance API in my case) until that chart container is blank for that time period. and it show up suddenly as data fetching is done.

so,
I wanted a **loading animation (busy indicator)** while data is not fetched yet inside the chart container. Like some other chart libraries provide. I also asked the same thing in the forums provided by Infragistics. here is the link for that question :  
[https://www.infragistics.com/community/forums/f/ignite-ui-for-react/125168/regarding-igrfinancialchart-loading](https://www.infragistics.com/community/forums/f/ignite-ui-for-react/125168/regarding-igrfinancialchart-loading)

I am attaching a video to clear the thing what I am trying to specify.

https://user-images.githubusercontent.com/86289427/226406152-b17cf552-4a34-45e3-bc52-fe7fe2c0d3ff.mp4

You can see when the data is not being fetched the chart shows loading animation and when data is fetched it shows the chart lines.
Basically I am asking for the same functionality inside the chart container itself so the name (i.e., title) of chart and other properties are being shown with animation until data is fetched.

let me know if there is any doubt or need more info in what I asked for.
